src/BasicLogAnalyser.php

```diff
@@ -678,9 +739,10 @@ private function createTables() {
 				"slug_id" integer not null,
 				"action" varchar(30) null,
 				"installed_version" varchar(30) null,
-				"wp_version" varchar(20) null,
+				"cms" varchar(20) null,
+				"cms_version" varchar(20) null,
 				"php_version" varchar(20) null,
-				"wp_version_aggregate" varchar(15) null,
+				"cms_version_aggregate" varchar(15) null,
```

Comment:
It would be nice if we could migrate existing data to the new schema so that people who were already using an older version of this project don't have to throw away their accumulated statistics. Thoughts?

Reply:
This seems to be a temporary table (is created with ':memory:'), it's necessary a data migration or some "alter table" ?

Reply:
Oops, you are correct. I forgot that it's an in-memory table. It's not necessary to migrate it.